services/wallet/reader.go

```diff
@@ -25,9 +26,10 @@ import (
 // WalletTickReload emitted every 15mn to reload the wallet balance and history
 const EventWalletTickReload walletevent.EventType = "wallet-tick-reload"
 const EventWalletTickCheckConnected walletevent.EventType = "wallet-tick-check-connected"
+const SIGNAL_MANUAL_RELOAD_TRIGGERED = "walletAccount/reset-wallet-reload-timer"
 
 const (
-	walletTickReloadPeriod      = 10 * time.Minute
+        walletTickReloadPeriod      = 3 * time.Minute
```

Comment:
Why 3?

Reply:
to speed up my tests, I will switch it back to 10 when I create the final PR

Reply:
just to speed up my tests.
fixed in final PR